lib/token-assert.js

```diff
@@ -38,6 +38,21 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
                 column: token.loc.end.column
             });
         }
+    } else if (options.hasOwnProperty('maximumSpaces')) {
```

Comment:
This is very close with https://github.com/jscs-dev/node-jscs/pull/1023/files#diff-526f88acb4a66447fc67eaf20ca6acadR104

We need to decide what naming we should use.
/cc @markelog @mdevils @mikesherov